### PR TITLE
Delete dependabot.yml

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Delete as is producing too much noise because cannot configure to only do security updates

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by zurbergram** [comment]
I can delete this if there will be an issue, the goal was to only get security updates so there less "gotchas" that require a separate PR by a dev

**Comment by philipwritescode** [comment]
> I can delete this if there will be an issue, the goal was to only get security updates so there less "gotchas" that require a separate PR by a dev

Dependabot is still generating separate PRs that need review.  For the `vets-api` we ended up not using this because it's just generally noiser than it was helpful

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,5 +11,3 @@ updates:
       interval: "daily"
       time: "09:00"
       timezone: "America/New_York"
-    # Allow up to 10 open pull requests for dependencies
-    open-pull-requests-limit: 10
```
Comment (erluti) [inline .github/dependabot.yml:15]: Is this using the default?  Is it 1?

**Review by philipwritescode — APPROVED** [review]
LGTM

File: .github/dependabot.yml (hunk 1 above)
Comment (zurbergram) [reply .github/dependabot.yml:15]: Default is 5
https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#open-pull-requests-limit

**Comment by zurbergram** [comment]
> > I can delete this if there will be an issue, the goal was to only get security updates so there less "gotchas" that require a separate PR by a dev
> 
> Dependabot is still generating separate PRs that need review. For the `vets-api` we ended up not using this because it's just generally noiser than it was helpful

Based on this i'm just going to delete this file

**Review by philipwritescode — APPROVED** [review]

**Pull request merged**